stage2/01-sys-tweaks/01-run.sh

```diff
@@ -21,6 +21,10 @@ else
 	systemctl disable ssh
 fi
 systemctl enable regenerate_ssh_host_keys
+if [ -z "${ROOT_PUBLIC_KEY}" ]; then
```

Comment:
shouldn't that be `-n`?

Reply:
You are right it could be better ;)

Fixed